app/proc/store.go

```diff
@@ -82,7 +82,7 @@ func (b BoltDB) Save(fmFeed string, item feed.Item) (bool, error) {
 }
 
 // Load from bold for given feed, up to max
-func (b BoltDB) Load(fmFeed string, max int) ([]feed.Item, error) {
+func (b BoltDB) Load(fmFeed string, max int, skipJunk bool) ([]feed.Item, error) {
```

Comment:
not sure I get it. Under what circumstances skipJunk false will make sense? I mean if we bothered to set up a filter it means we want to apply it.

Reply:
We need to load "junks" for UI. Method `Load` serve both endpoints - rss and feed-page. For page we need to call them with `skipJunk==false`.

Reply:
ok, probably I'd do this logic (include/exclude) outside of the Load, but your way is ok too.